modules/prometheus-exporter_oracle/conf/04-blockingsessions.yaml

```diff
@@ -0,0 +1,18 @@
+module: oracle
+name: "Blocking(s) session(s)"
+id: blocking_sessions
+transformation: true
+aggregation: ".mean(by=['job'])"
+
+signals:
+  signal:
+    metric: oracledb_blocking_sessions_count
+    rollup: latest
+
+rules:
+  critical:
+    threshold: 0
+    comparator: ">"
+    lasting_duration: "5m"
```

Comment:
be careful, you will not get any alert if a session does not remain continuously blocked during 5m.
I think having a block is not normal, no matter the time no?